docs/source/users.rst

```diff
@@ -13,24 +33,38 @@ When a customer is created, a user is also created associated with the customer.
 - Deleting providers
 
 Adding and Removing Users
```

Comment:
Should be removed. There isn't a customer owner or APIs to add/remove users. User creation happens on a valid request.